docs/getting-started.mdx

```diff
@@ -98,7 +95,7 @@ If [sharing privately](./concepts/sharing-private.mdx), only users with the shar
 
 ## Enabling Your zrok Environment
 
-After you have [an account](#get-an-account), you can enable your `zrok` environment.
+After you have [an account](https://myzrok.io/), you can enable your `zrok` environment.
```

Comment:
It doesn't make sense to link to myzrok.io here. The purpose of the get-an-account section was to present the user with the essential choice between myzrok and self-hosting.

Reply:
Linked back to the top section with the two button options